applications/subswap/contract/README.md

```diff
@@ -1,32 +1,42 @@
-# Subswap contract
+# Subswap Contract Deployment Guide
 
-## Subswap Deployment Guide
+## Configuration: `deploy.config.json`
 
-### Check `deploy.config.json` File
-
-1. Open the `deploy.config.json` file.
-2. Check the values of `parentnetendpoint` and `subnetendpoint`.
+1. **Open the `deploy.config.json` File**
+   - Verify the values for `parentnetendpoint` and `subnetendpoint`.
    - `parentnetendpoint`: The endpoint contract address on Parentnet.
    - `subnetendpoint`: The endpoint contract address on Subnet.
+   - `subnettoken`:
+     - `name`: The name of the token.
+     - `symbol`: The symbol of the token.
+     - `initSupply`: The initial supply of the token.
 
-### Deploy Subswap
+## Deploying Subswap
 
-#### Method 1: Using Hardhat Scripts
+### Method 1: Using Hardhat Scripts
 
-1. Install Hardhat and dependencies:
+1. **Install Hardhat and Dependencies**
 
-```
-yarn
-```
+   ```bash
+   yarn
+   ```
 
-2. Deploy the Parentnet Treasury contract:
+2. **Deploy the Parentnet Treasury Contract**
 
-```
-npx hardhat run scripts/parentnettreasurydeploy.js --network xdcparentnet
-```
+   ```bash
+   npx hardhat run scripts/parentnettreasurydeploy.js --network xdcparentnet
+   ```
 
-3. Deploy the Subnet Treasury contract:
+3. **Deploy the Subnet Treasury Contract**
 
+   ```bash
+   npx hardhat run scripts/subnettreasurydeploy.js --network xdcsubnet
+   ```
+
+## Deploying the Subnet Token
+
+```bash
+npx hardhat run scripts/simpletokendeploy.js --network xdcsubnet
 ```
-npx hardhat run scripts/subnettreasurydeploy.js --network xdcsubnet
-```
+
+This guide provides the steps to check and configure necessary values in the `deploy.config.json` file, and then deploy Subswap and the Subnet Token using Hardhat scripts.
```

Comment:
shouldn't this be at the top of the file

Reply:
done